src/go/rpk/pkg/cli/debug/bundle/bundle.go

```diff
@@ -268,9 +269,9 @@ BARE-METAL
 
  - Disk usage: The disk usage for the data directory, as output by 'du'.
 
- - redpanda logs: The node's redpanda logs written to journald. If --logs-since 
-   or --logs-until are passed, then only the logs within the resulting time frame
-   will be included.
+ - redpanda logs: The node's redpanda logs written to journald since yesterday. 
+   If --logs-since or --logs-until are passed, then only the logs within the
+   resulting time frame will be included.
```

Comment:
```suggestion
 - Redpanda logs: The node's Redpanda logs written to journald since yesterday. 
   If `--logs-since` or `--logs-until` are passed, then only the logs within the
   resulting time frame are included.
```

Reply:
@r-vasquez What does "since yesterday" mean? Is it the past 24 hours? If not, how does it decide what is yesterday vs today with different time zones?

Reply:
Fixed, thanks.

Though I had to change **`** for **'** since I couldn't use it in the current go format style that we have, hope it's ok.

Reply:
> What does "since yesterday" mean? Is it the past 24 hours? If not, how does it decide what is yesterday vs today with different time zones?

From systemd.time (what we use for this command): 
"today", "yesterday", and "tomorrow" refer to 00:00:00 of the current day, the day before, or the next day, respectively.

Should we add it to this specific paragraph (please note that tomorrow does not apply here)

Reply:
Added an additional message in https://github.com/redpanda-data/redpanda/compare/9396e476f542beeb0cd4ead7d2f0f17e268f349f..1b88856326f06d85b2f65db262bc99b2ce91c450